modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef OPENCV_GAPI_RENDER_HPP
+#define OPENCV_GAPI_RENDER_HPP
+
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi/opencv_includes.hpp>
+#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/own/exports.hpp>
+#include <opencv2/gapi/own/scalar.hpp>
+
+namespace cv
+{
+namespace gapi
+{
+
```

Comment:
Should it be in some new "osd" namespace?

Reply:
let it be `draw`. `cv::gapi::draw::Something`. Nice, isn't it? :)

Reply:
Or may be even `cv::gapi::wip::draw` - to make it solid with async API.

Reply:
Fixed